src/main/java/me/zeroeightsix/kami/module/modules/movement/ElytraFlight.kt

```diff
@@ -157,17 +159,17 @@ class ElytraFlight : Module() {
                 } else {
                     val calcMotionDiff = motionAmount * 0.008
                     mc.player.motionY += calcMotionDiff * 3.2
-                    mc.player.motionX -= (-MathHelper.sin(yaw)).toDouble() * calcMotionDiff / 1.0
-                    mc.player.motionZ -= MathHelper.cos(yaw).toDouble() * calcMotionDiff / 1.0
+                    mc.player.motionX -= (-sin(yaw)) * calcMotionDiff / 1.0
+                    mc.player.motionZ -= cos(yaw) * calcMotionDiff / 1.0
 
                     mc.player.motionX *= 0.99
                     mc.player.motionY *= 0.98
                     mc.player.motionZ *= 0.99
                 }
-            } else { /* runs when pressing wasd */
-                mc.player.motionX = (-MathHelper.sin(yaw)).toDouble() * speedControl.value
-                mc.player.motionY = (-fallSpeedControl.value).toDouble()
-                mc.player.motionZ = MathHelper.cos(yaw).toDouble() * speedControl.value
+            } else {/* runs when pressing wasd */
+                    mc.player.motionX = (-sin(yaw)) * speedControl.value
+                    mc.player.motionY = (-fallSpeedControl.value).toDouble()
+                    mc.player.motionZ = cos(yaw) * speedControl.value
```

Comment:
Did you mean to add this extra indentation?

Reply:
That was unintentional